web/src/components/navigation.js

```diff
@@ -38,27 +39,50 @@ const NavigationLink = styled.a`
 
 
 class Navigation extends Component {
+ 
   render() {
     return (
+      <Router>
       <NavigationList>
         <NavigationListItem>
-          <NavigationLink href="/">Dashboards</NavigationLink>
+            <NavigationLink href="/">
+                <Link to="/">Dashboards</Link>
+            </NavigationLink>
           </NavigationListItem>
         <NavigationListItem>
-          <NavigationLink href="/coops">Co-Ops</NavigationLink>
+            <NavigationLink href="/coops">
+                <Link to="/coops/">Co-Ops</Link>
+            </NavigationLink>
           </NavigationListItem>
         <NavigationListItem>
-          <NavigationLink href="/members">Members</NavigationLink>
+            <NavigationLink href="/members">
+                <Link to="/members/">Members</Link>
+            </NavigationLink>
           </NavigationListItem>
         <NavigationListItem>
-          <NavigationLink href="/transactions">Transactions</NavigationLink>
+            <NavigationLink href="/transactions">
+                <Link to="/transactions/">Transactions</Link>
+            </NavigationLink>
           </NavigationListItem>
         <NavigationListItem>
-          <NavigationLink href="/loans">Loans</NavigationLink>
+            <NavigationLink href="/loans">
+                <Link to="/loans/">Loans</Link>
```

Comment:
we should be able to stick to the original `<NavigationLink href="/loans">Loans</NavigationLink>` for these because the `<NavigationLink>` is covering the function of the `Link to`